### PR TITLE
Add support for bill, in-House group, member and question of the House of Councillors of Japan

### LINKED ISSUE #143: Add support for bill of the House of Councillors of Japan
https://github.com/smartnews-smri/house-of-councillors

### PR DESCRIPTION
**Pull request by tikkss**
Fixes #143.

### CODE REVIEW

**Comment by tikkss** [comment]
Sorry, I've rebased my branch.

**Comment by kou** [comment]
Could you rebase on master to fix the CI failures?

**Comment by tikkss** [comment]
Thanks! I've rebased on master and update expected, so the CI is successfully.

**Review by kou — APPROVED** [review]

File: test/test-house-of-councillor.rb [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                     247,
+                     record("足立　敏之",
+                            nil,
+                            "https://www.sangiin.go.jp/japanese/joho1/kousei/giin/profile/7016001.htm",
+                            "あだち　としゆき",
+                            "自民",
+                            "比例",
+                            Date.parse("2028-07-25"),
+                            "https://www.sangiin.go.jp/japanese/joho1/kousei/giin/photo/g7016001.jpg",
+                            "2016、2022",
```
[the remark this answers is not in the document]
Comment (tikkss) [reply test/test-house-of-councillor.rb:185]: Make sense. Let me try.

**Comment by kou** [comment]
Thanks!

**Pull request merged**